.cloudbuild.yaml

```diff
@@ -97,8 +97,12 @@ steps:
   id: 'buildDeployer'
 - name: 'gcr.io/cloud-builders/docker'
   entrypoint: '/bin/bash'
-  args: ['-c', 'cd /workspace/components/kubeflow/launcher && ./build_image.sh -p $PROJECT_ID -t $COMMIT_SHA']
+  args: ['-c', 'cd /workspace/components/kubeflow/tf-launcher && ./build_image.sh -p $PROJECT_ID -t $COMMIT_SHA']
```

Comment:
Is the change from launcher to tf-launcher necessary? It seems unrelated to this PR.

Reply:
I just rename launcher (in fact, it only launches tfjob) to tf-launcher since we can add more launchers (saying pytorch job launcher, studyjob launcher and so on)